docs/make.jl

```diff
@@ -1,27 +1,34 @@
+using Pkg
+
+# targeting the correct source code
+# this asumes the make.jl script is located in QEDcore.jl/docs
+project_path = Base.Filesystem.joinpath(Base.Filesystem.dirname(Base.source_path()), "..")
+Pkg.develop(; path=project_path)
+
 using LorentzVectorBase
 using Documenter
 
 DocMeta.setdocmeta!(
   LorentzVectorBase, :DocTestSetup, :(using LorentzVectorBase); recursive=true
 )
 
-const numbered_pages = [
-  file for file in readdir(joinpath(@__DIR__, "src")) if
-  file != "index.md" && splitext(file)[2] == ".md"
+pages = [
+  "Home" => "index.md", "Interface" => "10-interface.md", "Reference" => "95-reference.md"
 ]
```

Comment:
I'd vote for automatic discovery of pages, but there is nothing wrong updating list manualy

Reply:
Since we don't have that many pages, I would like to go for more control. Furthermore, it seems like something is broken with this in the current implementation, e.g. the local build does not link the pages correctly.